cinema/serializers.py

```diff
@@ -63,12 +93,15 @@ class Meta:
 class MovieSessionListSerializer(MovieSessionSerializer):
     movie_title = serializers.CharField(source="movie.title", read_only=True)
     cinema_hall_name = serializers.CharField(
-        source="cinema_hall.name", read_only=True
+        source="cinema_hall.name",
+        read_only=True
     )
+
     cinema_hall_capacity = serializers.IntegerField(
         source="cinema_hall.capacity", read_only=True
     )
     tickets_available = serializers.IntegerField(read_only=True)
+    movie_image = serializers.ImageField(source="movie.image", read_only=True)
```

Comment:
Code Efficiency: Good job on using `serializers.ImageField` for the movie image field instead of `serializers.CharField`. This is the correct way to handle image fields in Django.

Reply:
??